src/main/java/wolox/training/controller/UserController.java

```diff
@@ -55,6 +61,12 @@ public UserController(UserRepository userRepository, BookRepository bookReposito
      * @throws UserNotFoundException if there is no user associated with that username
      */
     @GetMapping("/username")
+    @ApiOperation(value = "Giving an username, return the user", response = User.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved user"),
+            @ApiResponse(code = 400, message = "Something went wrong"),
+            @ApiResponse(code = 404, message = "The resource you were trying to reach is not found"),
+            @ApiResponse(code = 500, message = "Internal Server Error")})
```

Comment:
Extract those string to constants. I should be a different file, accessible by every controller